core/controllers/classroom_test.py

```diff
@@ -348,8 +350,8 @@ def test_get_multiple_classrooms_count_all_published_unmatching_classrooms(
             **{'is_published': False}
         )
 
-        # Should return 'test_id' class, but count all public_classrooms,
-        # 'science' and 'test_id'.
+        # Should return classroom with 'test_id', but count all
+        # public_classrooms('science' and 'test_id').
```

Comment:
Minor nit: please add a space after "classrooms".

Reply:
Got it, thanks for pointing that out. I've made the update.